src/lib/data/buyables/titheFarmBuyables.ts

```diff
@@ -65,6 +65,30 @@ const TitheFarmBuyables: TitheFarmBuyable[] = [
 		},
 		titheFarmPoints: 5
 	},
+	{
+		name: `Gricoller's can`,
+		aliases: ['gricollers can'],
+		outputItems: {
+			[itemID(`Gricoller's can`)]: 1
+		},
+		titheFarmPoints: 200
+	},
+	{
+		name: `Seed box`,
+		aliases: ['seed box'],
+		outputItems: {
+			[itemID(`Seed box`)]: 1
+		},
+		titheFarmPoints: 250
+	},
+	{
+		name: `Herb sack`,
+		aliases: ['herb sack'],
+		outputItems: {
+			[itemID(`Herb sack`)]: 1
+		},
+		titheFarmPoints: 250
+	},
```

Comment:
If the buyable is just simple like these, you dont need the `outputItems` at all, it will default to 1x of the item in the name

Reply:
Was just trying to keep it internally consistent with Andre’s code.